src/cloudai/workloads/ai_dynamo/slurm_command_gen_strategy.py

```diff
@@ -27,10 +27,33 @@ class AIDynamoSlurmCommandGenStrategy(SlurmCommandGenStrategy):
 
     def _container_mounts(self) -> list[str]:
         td = cast(AIDynamoTestDefinition, self.test_run.test.test_definition)
+
+        dynamo_repo_path = td.dynamo_repo.installed_path
+        if dynamo_repo_path is None:
+            raise ValueError("dynamo_repo_path is not set - repo may not be installed")
+        dynamo_repo_path = dynamo_repo_path.absolute()
+
         mounts = [
-            f"{td.huggingface_home_host_path}:{td.cmd_args.huggingface_home_container_path}",
+            f"{dynamo_repo_path}:{dynamo_repo_path}",
+            f"{td.cmd_args.huggingface_home_host_path}:{td.cmd_args.huggingface_home_container_path}"
+            if td.cmd_args.dynamo.backend == "vllm"
+            else f"{td.cmd_args.huggingface_home_host_path}:{td.cmd_args.huggingface_home_host_path}",
             f"{td.script.installed_path.absolute()!s}:{td.script.installed_path.absolute()!s}",
         ]
+
+        if td.cmd_args.dynamo.backend == "sglang":
+            deepep_path = (
+                dynamo_repo_path / "components/backends/sglang/configs/deepseek_r1/wideep/deepep.json"
+            ).absolute()
+            sgl_http_server_path = (
+                dynamo_repo_path / "components/backends/sglang/src/dynamo/sglang/utils/sgl_http_server.py"
+            ).absolute()
+            mounts.extend(
+                [
+                    f"{deepep_path!s}:{deepep_path!s}",
+                    f"{sgl_http_server_path!s}:{sgl_http_server_path!s}",
+                ]
+            )
```

Comment:
This should be removed from here and moved to toml itself where we use extra_container_mounts to mount the entire dynamo_repo into the container. We can then have additional parameters in the cmd_args section to specify paths for specific files, etc.